.github/vale-styles/Yugabyte/spelling-exceptions.txt

```diff
@@ -148,6 +148,7 @@ Datadog
 datasource
 datasources
 datetime
+datatypes
```

Comment:
actually it's data type(s) - 2 words. existing uses of datatype should gradually be expunged